L/Libtask/bundled/task.c

```diff
@@ -0,0 +1,91 @@
+/*
+ task.c
+ task copying (aka continuation) for lightweight processes (symmetric coroutines)
+ */
+
+#include "julia.h"
+
+// Workaround for JuliaLang/julia#32812
```

Comment:
OK, so I read this as saying: "JuliaLang/julia#32812 is fixing something we need but it is not available in all Julia versions so we work around this"; but you are really saying "JuliaLang/julia#32812 introduced a regression that we are working around". But where is the Julia issue which reports the regression and discusses solutions?